client/src/ui/molecules/breadcrumbs/index.scss

```diff
@@ -30,6 +30,7 @@
         mask-size: 12px;
         transform: rotate(-90deg);
         width: 12px;
+        flex-shrink: 0;
```

Comment:
Could you move this between `display` and `height` to keep the (mostly) alphabetical order here?

Reply:
sure. should I also update stylelint configuration? it didn't throw any warning about this.

Reply:
Good idea, but could you open a separate PR for this?